test-chromium.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#
+# Build the extension and run a chromium extension with it installed.
+#
+set -o errexit -o xtrace
+
+source makecrx.sh
+
+# If you just want to run Chromium with the latest code:
+if [ "$1" == "--justrun" ]; then
```

Comment:
Move this above makecrx and add a `shift` here so the `--justrun` doesn't get passed to makecrx.